### PR TITLE
LXC installation and configuration improvements

### PR DESCRIPTION
**Pull request by Mopster**
* Refactored and improved the installation scripts of the LXC's in the KVM. The scripts now reside in the 'common' folder.
* Updated CentOS distro version to 6.6
* Rewrote the README
* Added installation of certain tools to the LXC :
  * iperf3
  * tcpdump
  * nc
  * strace

### CODE REVIEW

File: kvm-guests/common/lxc/iperf3.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+#
+# requires:
+#  bash
+#
+set -e
+set -o pipefail
+
+chroot_dir=${1}
+lxc_suffix=${2}
+file=iperf3-3.0.10-1.el6.x86_64.rpm
```
Comment (hansode) [inline kvm-guests/common/lxc/iperf3.sh]: Specified version looks bad.
EPEL yum repositry's policy is removing old version.
if iperf package released, that version package will be removed.

File: kvm-guests/common/lxc/iperf3.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+if [ ! -f ${file} ]; then
+  chroot ${chroot_dir} /bin/bash -ex <<EOS
+    wget http://dl.fedoraproject.org/pub/epel/6/x86_64/${file}
+EOS
+fi
+
+chroot ${chroot_dir} /bin/bash -ex <<EOS
+  rpm -i --root=/var/lib/lxc/lxc${lxc_suffix}/rootfs --nodeps ${file}
```
Comment (hansode) [inline kvm-guests/common/lxc/iperf3.sh]: why don't you use "yum install iperf" ?

Comment (Mopster) [inline kvm-guests/common/lxc/iperf3.sh]: it doesn't work when using --installroot, it won't find the repository. I also tried with chroot, but the same propblem persisted, this was the only way I could find to install iperf3 on the LXC

Comment (hansode) [inline kvm-guests/common/lxc/iperf3.sh]: ok. thx.

**Comment by hansode** [comment]
:+1:

**Comment by Mopster** [comment]
I think it's ok to merge now ? :-)

**Comment by Metallion** [comment]
Since you added a commit after the last +1, I think you need a new one. This is mostly ham-san's project so he should give one.

**Comment by Mopster** [comment]
Yeah, (almost) never merge your own Pull Requests ;-)

**Comment by hansode** [comment]
LGTM :+1:

**Pull request merged**